src/script/bitcoinconsensus.h

```diff
@@ -48,16 +48,34 @@ typedef enum bitcoinconsensus_error_t
-                                                               bitcoinconsensus_SCRIPT_FLAGS_VERIFY_NULLDUMMY | bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CHECKLOCKTIMEVERIFY |
-                                                               bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CHECKSEQUENCEVERIFY | bitcoinconsensus_SCRIPT_FLAGS_VERIFY_WITNESS
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_NONE                                  = 0,
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_P2SH                                  = (1U << 0), // evaluate P2SH (BIP16) subscripts
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_STRICTENC                             = (1U << 1), // enforce strict adhererence to pubkey/sig encoding standards
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_DERSIG                                = (1U << 2), // enforce strict DER (BIP66) compliance
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_LOW_S                                 = (1U << 3), // enforce low-S value in ECDSA sig (BIP 62 rule 5)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_NULLDUMMY                             = (1U << 4), // enforce NULLDUMMY (BIP147)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_SIGPUSHONLY                           = (1U << 5), // enforce non-push ops in scriptSig (BIP 62 rule 2)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_MINIMALDATA                           = (1U << 6), // enforce minimal push (BIP 62 rules 3 and 4)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_DISCOURAGE_UPGRADABLE_NOPS            = (1U << 7), // discourage use of reserved NOPs for upgrades
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CLEANSTACK                            = (1U << 8), // enforce non-superfluous scriptSig ops (BIP 62 rule 6)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CHECKLOCKTIMEVERIFY                   = (1U << 9), // enable CHECKLOCKTIMEVERIFY (BIP65)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CHECKSEQUENCEVERIFY                   = (1U << 10), // enable CHECKSEQUENCEVERIFY (BIP112)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_WITNESS                               = (1U << 11), // enable WITNESS (BIP141)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_DISCOURAGE_UPGRADABLE_WITNESS_PROGRAM = (1U << 12), // discourage use of v1-v16 witness program
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_MINIMALIF                             = (1U << 13), // enforce MINIMALIF (BIP342)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_NULLFAIL                              = (1U << 14), // enforce NULLFAIL (BIP146)
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_WITNESS_PUBKEYTYPE                    = (1U << 15), // enforce compression of pubkey in segwit scripts
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_CONST_SCRIPTCODE                      = (1U << 16), // enforce constant scriptCode policy in non-segwit scripts
+    bitcoinconsensus_SCRIPT_FLAGS_VERIFY_ALL                                   = bitcoinconsensus_SCRIPT_FLAGS_VERIFY_P2SH | bitcoinconsensus_SCRIPT_FLAGS_VERIFY_DERSIG |
```

Comment:
In commit "[policy] Extend bitcoinconsensus library with standard script flags" (e07ecdedc70ea045f98b592d36fdbf543696db28)

It seems like the name `VERIFY_ALL` is going to be misleading if it isn't actually going to include every verify flag. Would maybe suggest calling it `VERIFY_CONSENSUS` and changing VERIFY_ALL into a deprecated alias for VERIFY_CONSENSUS

Reply:
I would like but this may break upstream libraries using exported flag?

Reply:
Not suggesting API bump. Suggestion is to use `VERIFY_CONSENSUS` instead of `VERIFY_ALL` in main definition for clarity in new code and add `VERIFY_ALL = VERIFY_CONSENSUS /* deprecated */` at the bottom for backwards compatibility with old code